pydis_site/templates/resources/resources.html

```diff
@@ -1,90 +1,199 @@
-                              </a>
-                          </div>
-                      </div>
-                  </div>
-                  <a class="tile is-parent" href="{% url "resources:resources" category="tools" %}">
-                      <article class="tile is-child box hero is-dark">
-                          <p class="title is-size-1"><i class="fad fa-tools" aria-hidden="true"></i> Tools</p>
-                          <p class="subtitle is-size-4">Things we love to use</p>
-                      </article>
-                  </a>
-              </div>
-          </div>
-      </div>
-  </section>
+    {% include "base/navbar.html" %}
+
+    <section class="section">
+        <div class="container">
+            <div class="content">
+                <h1 class="resource-title has-text-centered">Resources</h1>
+                <hr/>
+            </div>
+            <div class="panel is-hidden-mobile">
+                <p class="panel-heading has-text-centered">Search Options</p>
+
+                <div class="field">
+                    <div class="columns">
+                        <div class="column pl-6 is-two-fifths is-flex is-flex-direction-column">
+                            <div class="title is-5 pt-2 has-text-centered">Topic</div>
+                            <div class="columns">
+                                <div class="column">
+                                    {% for topic in topics_1 %}
+                                    <div class="field">
+                                        <label class="checkbox">
+                                            <input class="topic" name="topic" type="checkbox" value="{{ topic }}"{% if topic == 'General' %} checked{% endif %}>
+                                            <span class="has-text-grey is-size-6">{{ topic }}</span>
+                                        </label>
+                                    </div>
+                                    {% endfor %}
+                                </div>
+                                <div class="column">
+                                    {% for topic in topics_2 %}
+                                    <div class="field">
+                                        <label class="checkbox">
+                                            <input class="topic" name="topic" type="checkbox" value="{{ topic }}"{% if topic == 'General' %} checked{% endif %}>
+                                            <span class="has-text-grey is-size-6">{{ topic }}</span>
+                                        </label>
+                                    </div>
+                                    {% endfor %}
+                                </div>
+                            </div>
+                            <span class="control ml-2 is-flex is-align-items-end is-justify-content-center mt-auto">
+                                <button onclick="selectAllQueryParams('topic')" class="button is-success is-small">Select All</button>
```

Comment:
This button does not seem to do anything if you click it when everything is already selected, is that intended?